resources/css/site.css

```diff
@@ -603,3 +603,49 @@ a.text-center.btn.btn-outline-light.btn__book {
 .card-img-top {
     height: 100% !important;
 }
+
+.p-heading {
+    font-weight: 500;
+}
+
+.p-container {
+    flex-direction: column-reverse;
+    padding: 40px;
```

Comment:
Generally we try to use `rem` for sizing, as it has a few minor benefits over using `px`.

As this project already uses `px` quite extensively, I wouldn't worry about modifying it here. But just for future reference in other projects (we tend to use a `rem()` mixin for SASS to make calculating this easier. But you can also generally calculate it without too many problems unless the values are not easily divisable. In this case it would be 40/16 = 2.5rem)

Reply:
Haven't used rems as followed previous code where everything has been set using px. But you're absolutely right.